projects/openapi-io/src/validation/validation-schemas.ts

```diff
@@ -1,3 +1,1574 @@
+export const basic_swagger2_object = {
+  type: 'object',
+  required: ['swagger', 'info', 'paths'],
+  properties: {
+    swagger: {
+      type: 'string',
+      enum: ['2.0'],
+    },
+    info: {
+      type: 'object',
+    },
+    paths: {
+      type: 'object',
+      patternProperties: {
+        '^/': {
+          type: 'object',
+          properties: {
+            get: {
+              $ref: '#/definitions/operation',
+            },
+            put: {
+              $ref: '#/definitions/operation',
+            },
+            post: {
+              $ref: '#/definitions/operation',
+            },
+            delete: {
+              $ref: '#/definitions/operation',
+            },
+            options: {
+              $ref: '#/definitions/operation',
+            },
+            head: {
+              $ref: '#/definitions/operation',
+            },
+            patch: {
+              $ref: '#/definitions/operation',
+            },
+          },
+        },
+      },
+    },
+  },
+  definitions: {
+    operation: {
```

Comment:
shouldn't this also have operationId produces consumes, parameters tags and summary?